src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.constants.Color;
+import lombok.Getter;
+
+@Getter
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        String info = "circle: area = "
+                + calculateArea()
+                + " sq.units, radius = "
+                + radius
+                + " units, area = "
+                + color.name().toLowerCase()
+                + '.';
+        System.out.println(info);
```

Comment:
```suggestion
        System.out.println("circle: area = "
                + calculateArea()
                + " sq.units, radius = "
                + radius
                + " units, area = "
                + color.name().toLowerCase()
                + '.');
```
there is no need to create a separate variable

Reply:
Done